force-app/main/default/lwc/productFilter/productFilter.html

```diff
@@ -35,10 +35,10 @@ <h1>Category</h1>
                     </template>
                 </template>
                 <template if:true={categories.error}>
-                    <c-inline-message
-                        message="Error loading categories"
+                    <c-error-panel
```

Comment:
Inline messages without illustrations may be more suited in this case. UX looks a bit weird if one or more of the filters fail to load:

![Screenshot 2020-04-16 at 15 10 54](https://user-images.githubusercontent.com/5071767/79460039-8e0a2a00-7ff4-11ea-9f5e-fcfbee6cd30f.png)

Consider adding a public property to hide the illustration? Have it visible by default and lets disable it where it makes sense.